Examples/Hyperspace-iOSExample/ViewController.swift

```diff
@@ -14,10 +14,33 @@ class ViewController: UIViewController {
     // MARK: - IBOutlets
     
     @IBOutlet private var postTextField: UITextField!
+    @IBOutlet private var serverTrustValidationToggle: UISwitch!
     
     // MARK: - Properties
     
     private let backendService = BackendService(networkService: NetworkService(networkActivityIndicatable: UIApplication.shared))
+    private lazy var trustValidatingBackendService: BackendService = {
+        // Creates a trust configuration whereby requests to the domain 'jsonplaceholder.typicode.com' will be validated to ensure they are being served a certificate matching 'jsonplaceholder.der'
+        let domainConfiguration = try? TrustConfiguration.DomainConfiguration(domain: "jsonplaceholder.typicode.com", certificates: certificate(named: "jsonplaceholder").map { [$0] } ?? [])
+        let validatingNetworkService = TrustValidatingNetworkService(trustConfiguration: domainConfiguration.map { [$0] } ?? [],
+                                                                     networkActivityIndicatable: UIApplication.shared)
+        return BackendService(networkService: validatingNetworkService)
+    }()
+    
+    var preferredBackendService: BackendService {
+        return serverTrustValidationToggle.isOn ? trustValidatingBackendService : backendService
+    }
+    
+    // MARK: - Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        //Create's a trust configuration whereby requests to the domain 'jsonplaceholder.typicode.com' will be validated to ensure they are being served a certificate matchin 'jsonplaceholder.der'
```

Comment:
The code here is a duplicate of the setup in the lazy property for `trustValidatingBackendService`